Backpack-SwiftUI/GraphicPromo/Classes/BPKGraphicPromo.swift

```diff
@@ -231,6 +248,17 @@ public struct BPKGraphicPromo: View {
         }
         return result
     }
+    
+    private func aspectRatio() -> CGFloat {
+        switch layoutType {
+        case .mobile:
+            return Constants.aspectRatioMobile
+        case .tablet:
+            return Constants.aspectRatioTablet
+        case .desktop:
```

Comment:
not sure this is actually needed? it does not hurt i guess though

Reply:
Good point! [Desktop](https://www.figma.com/design/4RUU95tmRl3SqzdJpmnT0F/Sponsored-Promo-v2?node-id=2-3982&t=UsKXRKMoBXGgJEAO-4) is for > 1024 points, which only really happens on iPad Pro 12.9".

We can actually go without it, but I think it does look slightly better and it's part of the [Figma Spec](https://www.figma.com/design/4RUU95tmRl3SqzdJpmnT0F/Sponsored-Promo-v2?node-id=2-3982&t=UsKXRKMoBXGgJEAO-4):


 With Desktop format:
![Simulator Screenshot - iPad Pro 13-inch (M4) - 2024-11-26 at 14 51 26](https://github.com/user-attachments/assets/9c2c0640-cc47-4bfd-8a36-78047723eb4e)

With Tablet format:

![Uploading Simulator Screenshot - iPad Pro 13-inch (M4) - 2024-11-26 at 14.52.25.png…]()